lib/src/delegates/asset_picker_viewer_builder_delegate.dart

```diff
@@ -175,6 +175,18 @@ abstract class AssetPickerViewerBuilderDelegate<Asset, Path> {
     );
   }
 
+  /// Call when the viewer is calling [State.didUpdateWidget].
+  /// 当预览器调用 [State.didUpdateWidget] 时操作 [State] 和 [TickerProvider]。
+  void didUpdateViewer(
+    covariant AssetPickerViewerState<Asset, Path> state,
+    covariant AssetPickerViewer<Asset, Path> oldWidget,
+    covariant AssetPickerViewer<Asset, Path> newWidget,
+  ) {
+    doubleTapAnimationController =
+        oldWidget.builder.doubleTapAnimationController;
```

Comment:
Should be `newWidget.builder.doubleTapAnimationController` instead?

Reply:
using `newWidget.builder.doubleTapAnimationController` or `state.builder.doubleTapAnimationController` throw the `LateInitializationError`.

Since `doubleTapAnimationController` is initialized at `initState`, i think it makes sense to reuse it in `didUpdateWidget` with `oldWidget`.